docs/about/index.rst

```diff
@@ -17,13 +40,156 @@ Code of conduct
+a relatively rare use case. They also discovered that fairness assessment is a
+common need, along with access to domain-specific guides to fairness metrics
+and unfairness mitigation algorithms. Additionally, many use cases take the
+form of regression or ranking, rather than classification. As a result of
+these insights, fairness assessment and use-case notebooks became key
+components of Fairlearn. Fairlearn also focuses on machine learning tasks
+beyond binary classification.
+
+The project was greatly expanded in the second half of 2019 thanks to the
+involvement of many contributors from Azure ML and Microsoft Research.
+At that time, the project started to have regular releases.
+
 .. _roadmap:
 
 Project roadmap
 ---------------
 
+As an open-source project, Fairlearn strives to incorporate the best of
+research and practice.  AI is a rapidly evolving field, and fairness in AI is
+all the more so. We therefore encourage researchers, practitioners, and other
+stakeholders to contribute fairness metrics, unfairness mitigation algorithms,
+and visualization capabilities to Fairlearn as we experiment, learn, and
+evolve the project together.
+
+There are many areas for future enhancement and growth. For example, Fairlearn
+currently supports only *group fairness*&mdash;i.e., fairness with respect to
+groups of people, such as those defined in terms of race, sex, age, or
+disability status&mdash;and not other conceptualizations of fairness, such as
+*individual fairness* or *counterfactual fairness.* Fairlearn also currently
+includes only a limited set of unfairness mitigation algorithms, although we
+note that these algorithms are not restricted to classification tasks. Besides
+adding fairness metrics, conceptualizations of fairness, and unfairness
+mitigation algorithms, we also hope that Fairlearn will expand to cover more
+complex machine learning tasks in areas like counterfactual reasoning,
+computer vision, and natural language processing.
+We also anticipate integrating Fairlearn with interpretability tools, such as
+`InterpretML <https://github.com/interpretml>`_.
+
+Ultimately, we hope that Fairlearn will become more than a software
+tool&mdash;a vibrant community and resource center that provides not only
+code, but also resources like domain-specific guides for when and when not to
+use different fairness metrics and unfairness mitigation algorithms.
+
+Short-term roadmap
+^^^^^^^^^^^^^^^^^^
+
+*Last update: June 19, 2020*
+
+Here we list some specific areas we would like to work on in the next few
+weeks and months. If you want to get involved, please
+:ref:`reach out <communication>`.  The inclusion of tasks in the list below
+does not mean that we are committing to deliver them by a specific date.
+It merely represents a current list of tasks that members of the community
+work on. If you are interested to work on a topic that is not covered, please
+reach out.
+
+Tasks
```

Comment:
The name for the documentation workstream is this general because it's two things:
- setting up the structure (there's a user guide, notebooks, quickstart, etc.) and utilities (sphinx, sphinx-gallery, etc.) for documentation to enable people to write useful documentation
- actually writing documentation to explain concepts or example scenarios

The latter should usually be counted in whatever workstream that adds the functionality, or in the notebooks workstream. However, we've added a lot of functionality in the past without writing all the documentation (user guides, for example) for it. So since these two things are very much related (you only figure out what you need in task 1 if you start with task 2) I've thrown them into a single workstream.

Taking a step back from the documentation workstream (which is just one such example), I totally agree that the "why" for each workstream should be clear. I'm not sure I can come up with concise names for each of them that explain "why". I'm not 100% sure whether you mean the "why" should be in the task title, or it's enough if it's clear from the task description.